### PR TITLE
IA-1953 Fix selenium tests

### PR DESCRIPTION
**Pull request by Qi77Qi**
Hope with https://github.com/broadinstitute/workbench-libs/pull/314, things will be fixed...

https://github.com/broadinstitute/firecloud-automated-testing/pull/119 is also needed...See https://github.com/seleniumhq/docker-selenium/releases

When tested locally, chromedriver 83.0.4103.97 works

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
jenkins retest...weird dependency not found error

**Comment by Qi77Qi** [comment]
weird error from sam in fiab-start...jenkins retest

**Comment by Qi77Qi** [comment]
fail to download sbt...jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala [hunk 1]
```diff
@@ -115,14 +114,15 @@ final class LeonardoSuite
 
 final class LeonardoTerraDockerSuite
     extends Suites(
-      new NotebookAouSpec,
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala:118]: Mentioned in slack.. do we know why these are failing? Is the test logic out of date with the current images or do we still have some dependency/selenium issues?

**Review by rtitle — APPROVED** [review]
Looks fine to me.. we should ticket fixing the commented specs since we should have test coverage for each supported terra-docker image ideally.

**Comment by Qi77Qi** [comment]
jenkins retest..fiab start failure

**Comment by Qi77Qi** [comment]
number of tests are now 34 compared to 15 before this PR...so going to merge this

**Pull request merged**